AuthBridge/AuthProxy/quickstart/k8s/demo-app-deployment.yaml

```diff
@@ -1,32 +1,32 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: auth-target
+  name: demo-app
```

Comment:
I think the old name `auth-target` was better, as it explained that this is the target service, where the token is being evaluated, instead of the main application that this token describes.